back/src/routes/airtable.ts

```diff
@@ -105,6 +105,7 @@ airtableRouter.post('/signout', function(req, res, next) {
 
 
 airtableRouter.post("/getInfo", function (req, res) {
+  console.log(JSON.stringify(req.body[0]))
```

Comment:
should this be in the pr?

Reply:
```suggestion
```

Reply:
removed that line, that was to sort out some of my confusion